h2/src/main/org/h2/command/Parser.java

```diff
@@ -7697,6 +7697,9 @@ private void readIfEqualOrTo() {
     }
 
     private Prepared parseSet() {
+        if (readIf("SESSION") || readIf("LOCAL")) {
+            // for PostgreSQL compatibility
+        }
```

Comment:
This code should be used only in PostgreSQL compatibility mode.

Reply:
And even in the PostgreSQL compatibility mode this code should not break `SET SESSION CHARACTERISTICS …`, this command is a part of the SQL Standard and it is supported by PostgreSQL too.

Reply:
removed from this PR